### ISSUE TITLE
File Sending Priorities [enhancement]

### ISSUE THREAD

**Issue by pwn0r** [syncthing/syncthing#4853]
It appears the sending priorities for syncthing are suboptimal.

I use smallest files first, normally I download stuff on my remote machine and then sync that folder with my home machine.  Small files are small (few MBs) and large files are large (few GBs) in my scenario. My actual priority is to have files completed. Have no use for incomplete files in my case. 

So in my case I shared first one subfolder with small files + one 4.3GB file. Then whilst the large file is being synced I added on the remote machine two more subfolders with 3GB and 4GB files. Then there is a connection break at my home side, by that time the first large file has been synced by 3GB.  After connection restored and syncthing restarted, I can see a funny picture -- it sends all the small files first outstanding, which is correct, and then sends me 3GB and 4GB files. For whatever reason only two files are being send simultaneously. Then only after 3GB file is completed syncthing suddenly discovers that 3gb of the 4.3gb file has been already uploaded and continues to sync it. 

So in short it appears only _actual_ file size as reported by the filesystem is taking into account for sending priorities. It should prioritize smallest size _outstanding_. 

### Version Information
(on both machines)

Syncthing Version: syncthing v0.14.46-rc.2 "Dysprosium Dragonfly" (go1.10 windows-amd64) 
OS Version: Windows 10 1607
Browser Version: (if applicable, for GUI issues)

**Comment by AudriusButkevicius** [comment]
Outstanding has no meaning, the file might have changed and be full of useless garbage for all that we know.


In the out of sync dialog you can click a button to bump files
